lib/modules/platform/azure/index.ts

```diff
@@ -383,6 +383,7 @@ const azureToRenovateStatusMapping: Record<GitStatusState, BranchStatus> = {
   [GitStatusState.NotApplicable]: 'green',
   [GitStatusState.NotSet]: 'yellow',
   [GitStatusState.Pending]: 'yellow',
+  [GitStatusState.PartiallySucceeded]: 'yellow',
```

Comment:
@viceice I wonder why TypeScript asks this mapping to be comprehensive, do you know why?